api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -2044,6 +2121,63 @@ components:
           description: Codename (technical) of this permission
           type: string
 
+    V1AreasGEOJsonList:
+      description: !!!EXPERIMENTAL!!! - Area GEOjson list
+      type: object
+      properties:
+        'type':
+          type: string
+          example: "FeatureCollection"
+        features:
+          type: array
+          items:
+            items:
+              $ref: '#/components/schemas/V1AreaGEOJson'
+
+    V1AreaGEOJson:
+      description: !!!EXPERIMENTAL!!! - Area GEOjson
+      type: object
+      properties:
+        'type':
+          type: string
+          example: "Feature"
+        geometry:
+          type: object
+          properties:
+            'type':
+              type: string
+              example: "MultiPolygon"
+            coordinates:
+              type: array
+              items:
+                type: integer
+              minItems: 0
+              maxItems: 2
+              example: [[[[0.0, 0.0], [0.0, 1.0], [1.0, 1.0], [0.0, 0.0]]], [[[1.0, 1.0], [1.0, 2.0], [2.0, 2.0], [1.0, 1.0]]]]
+        'properties':
+          type: object
+          properties:
+            id:
```

Comment:
Add to the description, that this ID cannot be used to retrieve the area instance.

Reply:
Removed the id from the response for now

Reply:
It is still in the documentation.

Reply:
Damn! 😂 forgot it in 1 place. I removed it from the swagger file.